content/src/controller/Controller.ts

```diff
@@ -370,6 +370,21 @@ export class Controller {
     return '?' + nextQueryParams
   }
 
+  async getActiveDeploymentByContentHash(req: express.Request, res: express.Response) {
```

Comment:
We should probably change all `getActiveDeploymentByContentHash` for `getActiveDeploymentsByContentHash`

Reply:
Renamed!